README.md

```diff
@@ -20,31 +20,16 @@ This project uses Paperclip gem which requires ImageMagick and `file` command. I
 ## Installation
 After prerequisites have been met, run the following commands:
 
-Download the repo from github by running in desired directory:
-```
-git clone <repo>
-```
-Navigate into your directory through terminal and install ruby gems by running the following command:
-```
-bundle install
-```
-Finally, finish setting up the environment by running the following commands:
-```
-rails db:migrate
-```
-```
-rails db:seed
-```
-In the ***config/application.yml*** file, add configuration values for website mailer to work. 
-
-Lastly, to launch the website, run:
-```
-rails server 
-```
-or
-```
-rails s 
-```
+1. Open terminal on your computer 
+2. Navigate to the directory where you would like to store the program
+3. Run the command ```git clone https://github.com/cs121-Unigon/FirstGenFacultyPage.git```
+4. Type ```cd FirstGenFacultyPage/FGFP``` to navigate to the correct directory 
+5. Type ```bundle install``` to install the necessary ruby gems 
+6. Set up the database by typing ```rails db:migrate``` 
+7. Populate the database by typing ```rails db:seed```
+8. In the ***FirstGenFacultyPage/FGFP/config/application.yml*** file, fill in required email credentails for website mailer to work
```

Comment:
Proofreading thing: credentials, not credentails.
Also, can we specify a little more what the email credentials should be?

Reply:
will do